sdl2promela/sdl/model.py

```diff
@@ -1338,10 +1336,13 @@ def __gather_inputs(self):
         for inputSignal in self.source.input_signals:
             input = Input()
             input.name = inputSignal["name"]
-            input.parameters = self.__get_input_parameters(input.name)
-            if len(input.parameters) > 1:
-                raise Exception(f"Too many parameters for {input.name}")
-            self.__assign_input_parameter_type(input, inputSignal)
+            # If signal has property "type" is has a parameter
+            # parameter name for signals is irrevelant for generation of promela
+            if "type" in inputSignal:
+                param = Parameter("p1")
```

Comment:
Why p1?

Reply:
The translator adds suffix `_input_param` so in the promela it is `p1_input_param`. I have no better idea

Reply:
Changed to input_param